### DISCUSSION TITLE
Offset Pointer/DragItem Positioning

### DISCUSSION THREAD

**Discussion by LinnJS** [clauderic/dnd-kit#1189] [Q&A]
I have the use case of needing to put a SortableList inside of a menu and the menu items have absolute positioning. In the video below you can see that there is a extra 64px in height from the button spacing. How would I go about subtracting 64px from the pointer drag positioning? 

Also you can see at the end of the clip if you just click on the handle the poising adjustment moves it 2 spaces.

I tried adjusting the transform on the DragItem and it felt weird. Do I need a custom Point sensor?

https://github.com/clauderic/dnd-kit/assets/23741323/7b088f44-d73f-4fb8-90fe-a031d5425b80

**Comment by LinnJS** [comment]
So what I think I might need is a "modifier" just not 100% sure the best place to make this modification.

**Reply by uncvrd** [reply]
Did you discover the solution? I just ran in to this with DnD in a modal :/
